package.json

```diff
@@ -26,9 +26,7 @@
     "@theforeman/vendor": ">= 6.0.0"
   },
   "dependencies": {
-    "c3": "^0.4.11",
-    "humanize-duration": "^3.20.1",
```

Comment:
This is being added in https://github.com/theforeman/foreman-js/pull/337
cc @MariaAga